coralproject/talk
# corl-2216-dont-submit-invalid-external-image

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2216 by creating a nested Form + Field (without rendering a nested <form> element) around the ExternalImageInput in order to verify the validity of the pasted image url before alerting the actual parent Form of it's presence. This addresses an edge case in which a user pastes an invalid image url, putting the MediaField form into an invalid state from which there is no exit.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
In a new comment, paste a Twitter link (or any value that is either not a valid URL or not a URL of an image) into the external media widget (accessed via the '+' button). It should alert you that this is an invalid image link. Then paste the same link in the body of the comment, at which point you should be presented with a prompt to attach the Tweet to your comment.
 
 
## How do we deploy this PR?
There shouldn't be any special considerations for deploying this PR.
